lib/util/include/hse/util/bloom_filter.h

```diff
@@ -71,7 +71,7 @@ bf_hash2bkt(u64 hash, u32 modulus, u32 bktshift)
     bit = (u32)(hash + (hash >> 32)) % modulus;
     bkt = bit >> bktshift;
 
-    return (bkt << bktshift) >> BYTE_SHIFT;
+    return (bkt << bktshift) >> BF_BYTE_SHIFT;
```

Comment:
This is the only place `BF_BYTE_SHIFT` is used.  Consider using local `const int shift = 3`  and removing the `#define`